docs/getting_started/requirements.md

```diff
@@ -15,123 +20,361 @@ Recommendation for production setups is to use Varnish/Fastly, Redis/Memcached,
 
-Optionally if you intend to edit [PNG, SVG, GIF or WEBP files in the Image Editor](images.md#image-optimization), or use it with image variations:
+    - JpegOptim
+    - Optipng
+    - Pngquant 2
+    - SVGO 1
+    - Gifsicle
+    - cwebp
 
-- JpegOptim
-- Optipng
-- Pngquant 2
-- SVGO 1
-- Gifsicle
-- cwebp
+=== "eZ Platform v2.5"
+
+    - GraphicsMagick
+    - ImageMagick
+    - GD
 
 ## [Clustering](clustering.md)
 
-- Linux NFS or S3/EFS (for IO, aka binary files stored in content repository, not supported with legacy)
-- Redis 4.0+, 5.0 or higher (separate instances for session and cache, both using a `volatile-*` [eviction policy](https://redis.io/docs/reference/eviction/), session instance configured for persistence) or [Memcached](https://memcached.org/) 1.5 or higher
-- [Varnish](http://varnish-cache.org/) 6.0LTS or 7.1 with [varnish-modules](https://github.com/varnish/varnish-modules/blob/master/README.md) or [Fastly](https://www.fastly.com/) using [the provided bundle](http_cache.md) (for HTTP Cache)
+=== "Ibexa DXP v4.5"
+
+    - Linux NFS or S3/EFS (for IO, aka binary files stored in content repository, not supported with legacy)
+    - Redis 4.0+, 5.0 or higher (separate instances for session and cache, both using a `volatile-*` [eviction policy](https://redis.io/docs/reference/eviction/), session instance configured for persistence) or [Memcached](https://memcached.org/) 1.5 or higher
+    - [Varnish](http://varnish-cache.org/) 6.0LTS or 7.1 with [varnish-modules](https://github.com/varnish/varnish-modules/blob/master/README.md) or [Fastly](https://www.fastly.com/) using [the provided bundle](http_cache.md) (for HTTP Cache)
+
+    If you see a "+" next to the product version, it indicates a recommended version or higher within the same major release. For example, "1.18+" means any 1.x version equal to or higher than 1.18, but not 2.x.
+
+=== "Ibexa DXP v3.3"
+
+    - Linux NFS or S3/EFS (for IO, aka binary files stored in content repository, not supported with legacy)
+    - Redis 4.0+, 5.0 or higher (separate instances for session and cache, both using a `volatile-*` [eviction policy](https://redis.io/docs/reference/eviction/), session instance configured for persistence) or [Memcached](https://memcached.org/) 1.5 or higher
+    - [Varnish](http://varnish-cache.org/) 6.0LTS with [varnish-modules](https://github.com/varnish/varnish-modules/blob/master/README.md) or [Fastly](https://www.fastly.com/) using [the provided bundle](https://doc.ibexa.co/en/3.3/guide/cache/http_cache/#serving-varnish-through-fastly) (for HTTP Cache)
+
+    If you see a "+" next to the product version, it indicates a recommended version or higher within the same major release. For example, "1.18+" means any 1.x version equal to or higher than 1.18, but not 2.x.
+
+=== "eZ Platform v2.5"
+
+    - php-redis 3.1.3+ or php-memcached 3.x+
```

Comment:
Isn't this a copy-paste glitch from "Cluster PHP packages" section?

Reply:
Yes, it is

```suggestion
    - Linux NFS or S3/EFS (for IO, aka binary files stored in content repository, not supported with legacy)
    - Redis 3.2+ (separate instances for session and cache, both using a `volatile-*` [eviction policy](https://redis.io/docs/reference/eviction/), session instance configured for persistence) or [Memcached](https://memcached.org/) 1.5 or higher
    - [Varnish](http://varnish-cache.org/) 5.1 or 6.0LTS with [varnish-modules](https://github.com/varnish/varnish-modules/blob/master/README.md) or [Fastly](https://www.fastly.com/) using [the provided bundle](https://doc.ibexa.co/en/2.5/guide/cache/http_cache/#serving-varnish-through-fastly) (for HTTP Cache)
```